content/en/docs/reference/terms/index.md

```diff
@@ -172,6 +172,8 @@ A FIDO2 [Discoverable Credential](#discoverable-credential) that can reliably be
 
 A [Passkey Provider](#passkey-provider) that plugs in to the OS via platform APIs to store and manage a user's passkeys via the platform authenticator.
 
+> NOTE: Some passkey providers intercept WebAuthn requests in order to provide support for passkeys using browser extensions which may bypass browser and/or platform interfaces and features.
```

Comment:
Suggesting some alternate wording:

```suggestion
> NOTE: Some passkey providers support passkeys via a browser extension that intercepts WebAuthn requests. This bypasses browser and/or platform interfaces and features.
```

Reply:
fine with me. @irew?

Reply:
While it does bypass, it doesn't mean the user is unable to access the browser/platform experience. That nuance should be captured. As an example, Dashlane will hand back to the browser/platform if the authenticator attachment is "cross-platform". Plus users are always able to fallback to the browser/platform if they need to.

Reply:
@irew What about something like this?

```suggestion
> NOTE: Some passkey providers support passkeys via a browser extension that intercepts WebAuthn requests. Providers that bypass browser and/or platform interfaces and features in this manner typically offer a way for the intercepted request to be passed back to the browser and/or platform to handle as usual.
```

Reply:
Lgtm 👍🏽